.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: Mark stale issues and pull requests
```

Comment:
Are we doing this for issues? It does not feel right if we do.

Reply:
Currently not, because it does not support both. However, it is not a bad idea in general.